services/consensuscontext/fetch_transactions.go

```diff
@@ -0,0 +1,29 @@
+package consensuscontext
+
+import (
+	"github.com/orbs-network/orbs-spec/types/go/services"
+	"time"
+	"fmt"
+)
+
+func (s *service) fetchTransactions(input *services.GetTransactionsForOrderingInput, minimumTransactionsInBlock int, belowMinimalBlockDelayMillis uint32) (*services.GetTransactionsForOrderingOutput, error) {
+
+	fmt.Println("Fetch transactions")
+	var proposedTransactions *services.GetTransactionsForOrderingOutput = nil
```

Comment:
why do we need to specifically define this? the line below will handle that, no? either case, the code should run the same without that line

Reply:
Right, fixed.